docs/prompts-for-creating-connectors.md

```diff
@@ -0,0 +1,37 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Creating a connector through prompts
+
+This guide contains prompts that help you create a connector using a set of prompts sent to the FortiAI's AI Assistant.
+
+1. Login to FortiSOAR.
+
+2. Navigate to **Content Hub**.
+
+3. Click to open the **Create** tab.
+
+4. Click the bot icon
+
+### Example: Create a Criminal IP connector
+
+* **Prompt**:
+
+>   1. _Build connector for Criminal IP product for following actions:_

```

Comment:
We will need more example prompts, their step by step user interaction along with screenshots and end result in create tab and in BYOC (BYOC details are good for only one example for rest we can skip) 
@amol can help here